ResearchKit/Common/ORKAnswerFormat_Internal.h

```diff
@@ -179,6 +179,15 @@ ORK_DESIGNATE_CODING_AND_SERIALIZATION_INITIALIZERS(ORKTimeIntervalAnswerFormat)
 
 @interface ORKTextAnswerFormat ()
 
+@property (nonatomic, assign, getter=isEmailAddress) BOOL emailAddress;
+
+@end
+
+
+@interface ORKEmailAnswerFormat ()
+
+@property (nonatomic, strong) ORKTextAnswerFormat *answerFormat;
```

Comment:
Prefer to declare `ORKTextAnswerFormat *_impliedAnswerFormat` in `ORKAnswerFormat.m`. Since this cached variable only need to be accessed by `ORKEmailAnswerFormat` itself.